plugins/catalog/api-report.md

```diff
@@ -319,15 +319,13 @@ export interface EntitySwitchProps {
   children: ReactNode;
 }
 
-// @public (undocumented)
-export function FilterContainer(props: {
-  children: React_2.ReactNode;
-}): JSX.Element;
+// @public @deprecated (undocumented)
+export const FilterContainer: (props: { children: ReactNode }) => JSX.Element;
 
-// @public (undocumented)
-export function FilteredEntityLayout(props: {
-  children: React_2.ReactNode;
-}): JSX.Element;
+// @public @deprecated (undocumented)
+export const FilteredEntityLayout: (props: {
+  children?: React.ReactNode;
```

Comment:
resulted in a new question mark on this - meh, that's ok

Reply:
👍, nah that should totally be fixed, was doing the cast wrong